test/plugin/applic-delayed-ckpt/Makefile

```diff
@@ -17,6 +17,7 @@ DMTCP_INCLUDE=${DMTCP_ROOT}/include
 
 CFLAGS += -I${DMTCP_INCLUDE} -DDMTCP -fPIC -c -g
 CXXFLAGS += -I${DMTCP_INCLUDE} -DDMTCP -fPIC -c -g
+LINKER = ${CC}
```

Comment:
s/LINKER/LINK ?

http://www.gnu.org/software/automake/manual/html_node/How-the-Linker-is-Chosen.html